cohttp-server-lwt-unix/bench/bench.ml

```diff
@@ -0,0 +1,33 @@
+open Lwt.Syntax
```

Comment:
Can you move this in https://github.com/mirage/ocaml-cohttp/tree/master/bench and update https://github.com/mirage/ocaml-cohttp/blob/master/bench/latency.sh and the dune file to have alsot the latency tests as part of the `cohttp-bench` package?